renderer/src/components/Onboarding.tsx

```diff
@@ -0,0 +1,108 @@
+import { FC, useState } from 'react'
+import { ReactComponent as Back } from './../assets/img/arrow-left.svg'
+import { ReactComponent as Page } from './../assets/img/paginator-page.svg'
+import { ReactComponent as CurrentPage } from './../assets/img/paginator-current.svg'
+
+interface IFooter {
+  page: number,
+  pages: number,
+  next: () => void,
+  prev: () => void
+}
+
+const Footer: FC<IFooter> = ({ page, pages, next, prev }) => {
```

Comment:
what do you think about naming interfaces like this `FooterProps`, to match what we already have in for example `FilAddressForm.tsx`?

```suggestion
interface FooterProps {
  page: number,
  pages: number,
  next: () => void,
  prev: () => void
}

const Footer: FC<FooterProps> = ({ page, pages, next, prev }) => {
```

I think in general this is a more common naming

Reply:
Similar on Onboarding component: 2e891618b82a949e5177ffcb11afbe6523e8d4c4